docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '3.7'
+
+services:
+
+  tomcat:
+    build: ./
+    container_name: tomcat-container
+    restart: always
+    environment:
+      JPDA_ADDRESS: "*:8000"
+      JPDA_TRANSPORT: dt_socket
```

Comment:
I believe `JPDA_TRANSPORT` env variable can be omitted, because socket connection is default for debugging.